network-api/networkapi/buyersguide/factory.py

```diff
@@ -364,39 +361,106 @@ class Meta:
     url = Faker('url')
 
 
+def create_general_product_visual_regression_product(seed, pni_homepage):
+    # There are no random fields here: *everything* is prespecified
+    GeneralProductPageFactory.create(
+        # page fields
+        title='General Percy Product',
+        first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        parent=pni_homepage,
+        # product fields
+        privacy_ding=True,
+        adult_content=True,
+        uses_wifi=True,
+        uses_bluetooth=True,
+        review_date=datetime(2025, 1, 1, tzinfo=timezone.utc),
```

Comment:
So, this doesn't actually seem to set the right date. @KalobTaulien any idea what the correct syntax would be?

Reply:
Ahh, review_date is using a DateField, whereas the others are using a DateTimeField().

```python
import datetime 
```

```suggestion
        review_date=datetime.date(2025, 1, 1),
```